### PR TITLE
[solution]: ko/easy-concat.md

### PR DESCRIPTION
**Pull request by kyeongsoosoo**
# Challenge [easy-concat]

## Checklist

- [x] Make sure that you are opening the PR with a single solution (one solution per PR)
- [x] The solution must have at least several references for the readers to dive in depth

### CODE REVIEW

File: ko/easy-concat.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+id: 533
+title: Concat
+lang: ko
+level: easy
+tags: array
+---
+
+## 챌린지
+
+타입 시스템 내에서 자바스크립트의 `Array.cocat` 함수를 구현해보세요.
```
Comment (ghaiklor) [inline ko/easy-concat.md:11]: Typo?

```suggestion
타입 시스템 내에서 자바스크립트의 `Array.concat` 함수를 구현해보세요.
```

**Review by ghaiklor — DISMISSED** [review]

File: ko/easy-concat.md (hunk 1 above)
Comment (kyeongsoosoo) [reply ko/easy-concat.md:11]: My mistake...😢

Comment (kyeongsoosoo) [reply ko/easy-concat.md]: @ghaiklor  I missed this suggestion. Could you please review it again?

**Review by ghaiklor — APPROVED** [review]

File: ko/easy-concat.md (hunk 1 above)
Comment (ghaiklor) [reply ko/easy-concat.md]: Done, thanks!

**Pull request merged**